README.adoc

```diff
@@ -1,25 +1,74 @@
 == Getting Started with Spring Cloud Task
-This guide walks you through an overview of Spring Cloud Task and the process of creating and launching short-lived
-batch application.
+This guide walks you through an overview of Spring Cloud Task and the process of creating and launching a short-lived application.
+
+== Starting with Spring Initializr
+
+You can use this https://start.spring.io/#!type=maven-project&language=java&platformVersion=3.0.0-RC2&packaging=jar&jvmVersion=17&groupId=com.example&artifactId=demo&name=demo&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.demo&dependencies=jdbc,cloud-task,mariadb,h2[pre-initialized project] and click Generate to download a ZIP file. This project is configured to fit the examples in this tutorial.
```

Comment:
Make `Generate` be bold. (The rule is that actions that one can take in a UI get bolded.)